commands/ls.go

```diff
@@ -47,6 +49,9 @@ type HostListItem struct {
 }
 
 func cmdLs(c CommandLine, api libmachine.API) error {
+	stateTimeoutDuration = time.Duration(c.Int("timeout")) * time.Second
+	log.Debugf("ls timeout set to %d seconds", stateTimeoutDuration/time.Second)
```

Comment:
I think we can just move the `var stateTimeoutDuration` into this function fully now and it no longer has to be global-level, no?